docs/using-cinder-csi-plugin.md

```diff
@@ -280,6 +280,35 @@ $ kubectl exec nginx -- df -h /var/lib/www/html
 Filesystem      Size  Used Avail Use% Mounted on
 /dev/vdb        2.0G   27M  1.97G   1% /var/lib/www/html
 ```
+## Inline Volumes
+This feature allows CSI volumes to be directly embedded in the Pod specification instead of a PersistentVolume. Volumes specified in this way are ephemeral and do not persist across Pod restarts. as of Kubernetes v1.16 this feature is beta so enabled by default. To enable this feature for CSI Driver, `volumeLifecycleModes` needs to be specified in CSIDriver object. The driver can run in `Persistent` mode, `Ephemeral` or in both modes. `podInfoOnMount` must be `true` to use this feature. 
```

Comment:
Just a nit. "As of Kubernetes v.1.16 ..."

Reply:
Link to sample driver-yaml in our manifest would be good to point out here. If only Ephermal or both is added volumeLifecycleModes , then only inline volumes feature is enabled right? Only persistent is there , then it is like inline disabled? If yes, could we make that explicit as well?

Reply:
@ramineni @chrigl Thanks for review, will update

Reply:
@ramineni @chrigl  Done!!